packages/authentication/test/poc/mock-express-router.js

```diff
@@ -0,0 +1,79 @@
+var fs = require('fs');
+var parser = require('url');
+var pathRegexp = require('path-to-regexp');
+
+var router = exports = module.exports = {};
+
+router.stack = [];
+
+router.route = function(path, method) {
```

Comment:
@bajtos the purpose here is not to use express. This POC checks for the dependencies of passport strategies on express. 
Here is what we achieved:

1. An Authentication class which can function independently of express
https://github.com/strongloop/loopback-next/blob/68c6732ee6fb0c71f3de6fc65d0b5c855f25acd0/packages/authentication/test/poc/mock-passport.js#L23

2. check the use of express constructs from passport like request.session, res.send

3. This initial poc addresses only passport-basic. I would use this to further test for oauth strategies.

Without a working model, how do you suggest to bridge passport strategies to loopback-next ?